server.go

```diff
@@ -1210,9 +1211,22 @@ type createSeriesIfNotExistsCommand struct {
 	Tags     map[string]string `json:"tags"`
 }
 
+type Point struct {
+	Name      string
+	Tags      map[string]string
+	Timestamp time.Time
+	Values    map[string]interface{}
+}
+
 // WriteSeries writes series data to the database.
 // Returns the messaging index the data was written to.
-func (s *Server) WriteSeries(database, retentionPolicy, name string, tags map[string]string, timestamp time.Time, values map[string]interface{}) (uint64, error) {
+func (s *Server) WriteSeries(database, retentionPolicy string, points ...Point) (uint64, error) {
```

Comment:
I don't have a strong preference toward `[]*Point` vs `[]Point` until we benchmark. It's less GC load if it's not pointers. It's much faster to sort if it is pointers. We'll just have to benchmark it later and see. No need to change anything right now.